### PR TITLE
#166240959 Users should be able to like and or dislike articles

### PR DESCRIPTION
**Pull request by EmyRukundo**
**What does this PR do?**

Users should be able to like and or dislike articles

**Description of task to be completed?**

Users should be able to like and or dislike articles when he or she are logged in

**How should this be manually tested?**

Cloning into this branch and trying to like and dislike an article

**What are the relevant pivotal tracker stories?**

#166240959

Screenshots
ezgif com-video-to-gif (2)

### CODE REVIEW

**Review by rukundoeric — APPROVED** [review]

**Comment by UhiriweAudace** [comment]
Kindly, @EmyRukundo  fix those Hound CI issues

File: src/redux/reducers/likeAndDislike.reducer.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+const initialState = {
+  likeArticle: {},
+  dislikeArticle: {},
+  getArticleLikes: {},
+  getArticleDislikes: {},
+};
+
+export default (state = initialState, { type, payload }) => {
+//   console.log('----', payload);
```
Comment (eliemugenzi) [inline src/redux/reducers/likeAndDislike.reducer.js]: Remove this comment

File: src/components/common/LikeAndDislike.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            <span className="text-danger">
+              <i className={`${this.state.isDisliked ? 'fas' : 'far'} fa-thumbs-down`} />
+            </span>
+            &nbsp;&nbsp;
+            {dislikes}
+          </Button>
+        </ButtonGroup>
+
+        {/* <div
+          onClick={() => this.handleLikeArticle(this.props.slug)}
```
Comment (eliemugenzi) [inline src/components/common/LikeAndDislike.js]: Remove these comments

**Review by eliemugenzi — CHANGES_REQUESTED** [review]

File: src/redux/reducers/likeAndDislike.reducer.js (hunk 1 above)
Comment (EmyRukundo) [reply src/redux/reducers/likeAndDislike.reducer.js]: @eliemugenzi , Done it, thanks

File: src/components/common/LikeAndDislike.js (hunk 2 above)
Comment (EmyRukundo) [reply src/components/common/LikeAndDislike.js]: corrected it

**Review by UhiriweAudace — APPROVED** [review]

**Review by eliemugenzi — APPROVED** [review]

**Review by deschantkn — APPROVED** [review]

**Pull request merged**